### DISCUSSION TITLE
After starting supabase locally: supabase studio is unhealthy

### DISCUSSION THREAD

**Discussion by jonny07** [supabase/supabase#21558] [Questions]
I just copied and started supabase with docker as explained here: https://supabase.com/docs/guides/self-hosting/docker
After "docker compose up -d" supabase studio is always unhealthy or latest after accessing localhost:3000 gets unhealthy.
I have a 4 core 8GB vserver with ubuntu 22.04.
I tried the current master branch with git clone and also the previous release [v0.23.11 (https://github.com/supabase/supabase/releases/tag/v0.23.11).
Any idea what I can do to leave supabase studio running healthy?
![image](https://github.com/supabase/supabase/assets/5730137/78a5ae4a-ecd4-479f-b4ad-8e4f8e20e5b1)
This is the start log: 
![image](https://github.com/supabase/supabase/assets/5730137/8e00ab7e-e725-4855-94c9-d9b0617729c6)
I just also tried it on a second vserver (6 cores, 12GB). It is the same problem. Should I use sth. else than ubuntu or no ARM? Or what can I do?

**Comment by bananasplit333** [comment]
I saw a solution somewhere but I can't seem to find it. Anyways, they said it was the webhost not binding to the localhost properly, resulting in an error. Try replacing the health check in your yaml file to : 


_require('http').get('http://' + process.env.HOSTNAME + ':3000/api/profile', (r) => {if (r.statusCode !== 200) throw new Error(r.statusCode)})_

**Reply by patzm** [reply]
#28105 is I think the culprit. The fix is also there, and in a linked PR.
